### ISSUE TITLE
Horizon Saber with 2 bottom parts

### ISSUE THREAD

**Issue by Ciampone** [keyteki/keyteki#4393]
I was playing a game and couldn't play my Horizon Saber because I'm with two bottom parts. I manually put both of the cards on the field and took a print
![error](https://github.com/user-attachments/assets/8e9d4ad0-81d3-49a9-a351-e1ed7ad0f385)

**Comment by strib** [comment]
Thanks.  This is a duplicate of #4382 and the fix is just waiting on a new deployment to the servers.

**Comment by strib** [comment]
(It's deployed now.)
